lib/paperclip/storage/google_drive.rb

```diff
@@ -156,7 +156,7 @@ def public_url_for title
       def search_for_title(title)
         parameters = {
                 'folderId' => find_public_folder,
-                'q' => "title contains '#{title}'", # full_title
```

Comment:
Since this method is a search, it would be convenient to find all `.pdf`, for example; or if you're storing file names with a prefix.

I have a feeling that the `contains` may have been chosen originally to match duplicate files?  @evinsou thoughts?